kokkos.presets.json

```diff
@@ -0,0 +1,58 @@
+{
+    "version"   : 3,
+    "configurePresets" : [
+        {
+            "name"          : "default",
+            "binaryDir"     : "${sourceDir}/build-with-${presetName}",
+            "cacheVariables" : {
+                "CMAKE_EXPORT_COMPILE_COMMANDS" : "ON",
```

Comment:
What is the point of exporting the compile command here?

Reply:
I guess it was needed mostly for me and myself in some early debugging phase. This isn't needed by default and you can consider it removed :smile: